nixos/modules/services/web-apps/c2FmZQ-server.nix

```diff
@@ -0,0 +1,138 @@
+{ lib, pkgs, config, ... }:
+
+let
+  inherit (lib) mkEnableOption mkPackageOption mkOption types;
+
+  cfg = config.services.c2FmZQ-server;
+
+  argsFormat = with lib; {
+    type = with types; let
+      valueType = nullOr (oneOf [
+        bool int str
+      ]) // {
+        description = "c2FmZQ-server params";
```

Comment:
I'd recommend against overriding this description. It's a lot more useful to know that this is a `nullOr (oneOf [ bool int str ])`, rather than having to go through the documentation upstream to figure out what values it supports.